.gitignore

```diff
@@ -61,4 +61,5 @@ typings/
 # next.js build output
 .next
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+.*.swp
```

Comment:
What is this?

Reply:
It makes git ignore temporary swap files vim creates while you're editing a file. Just a tiny quality of life thing.

Reply:
Oh, vim... 🤓